aries-site/src/pages/design/index.js

```diff
@@ -27,7 +27,58 @@ const Design = () => {
       {page.pages.length ? (
         <NavPage items={page.pages} topic={page.name.toLowerCase()} />
       ) : (
-        <ComingSoon />
+        <>
+          <ContentSection>
+            <Subsection name="Preferred">
+              <SubsectionText>
+                The HPE Design System is primarily focused on user interfaces
+                developed with ReactJS and Grommet. There are more resources,
```

Comment:
Maybe change to "We encourage teams to use ReactJS and Grommet if possible because there are more resources, examples, and community help for this environment."